app/process/watcher/crypto-transfer/crypto-transfer.go

```diff
@@ -1,3 +1,63 @@
-package crypto_transfer
+package cryptotransfer
 
-// crypto transfer watcher
+import (
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera"
+	cryptotransfermessage "github.com/limechain/hedera-eth-bridge-validator/app/process/model/crypto-transfer-message"
+	"github.com/limechain/hedera-eth-bridge-validator/app/process/watcher/publisher"
+	"github.com/limechain/hedera-watcher-sdk/queue"
+	"log"
+	"strconv"
+	"time"
+)
+
+type CryptoTransferWatcher struct {
+	client          *hedera.Client
+	accountID       hederasdk.AccountID
+	typeMessage     string
+	pollingInterval time.Duration
+}
+
+func (ctw CryptoTransferWatcher) Watch(queue *queue.Queue) {
+	go ctw.beginWatching(ctw.accountID, ctw.typeMessage, queue)
+}
+
+func (ctw CryptoTransferWatcher) beginWatching(account hederasdk.AccountID, typeMessage string, q *queue.Queue) {
+	lastObservedTimestamp := strconv.FormatInt(time.Now().Unix(), 10)
```

Comment:
Is this something temporary, until we have Database. If so a comment would be good

Reply:
Yes. It is temporary